client/my-sites/sharing/connections/connection.jsx

```diff
@@ -170,4 +186,13 @@ module.exports = React.createClass( {
 			</li>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, { connection } ) => ( {
+		connectionUser: getUser( state, connection.keyring_connection_user_ID ),
+		site: getSelectedSite( state ),
+		user: getCurrentUser( state ),
```

Comment:
Since we only use `user.ID`, should we just send `userId: getCurrentUserId( state )` to keep props simpler? Avoids a few `this.props.user && this.props.user.ID` conditions.